src/encoded/types/file.py

```diff
@@ -347,7 +347,7 @@ def _build_file_embedded_list() -> List[str]:
 
 @abstract_collection(
     name="files",
-    unique_key='accession',
+    unique_key="submitted_id",  # To permit lookup on submission
```

Comment:
Is this change why the above fix is needed for accession?

Reply:
This was necessary to allow the use of `submitted_id` for generic File linkTos. It exposed the accession problem to me, but isn't the sole reason for the fix.